src/app/index.js

```diff
@@ -6,10 +6,15 @@ import BookDetail from './screens/BookDetail';
 import Dummy from './screens/Dummy';
 
 import * as Routes from '../constants/routes';
-import { defaultNavigationOptions, bookDetailNavigationOptions, bookListNavigationOptions, dummyNavigationOptions, childTabsNavigationOptions } from '../config/navigationOptions';
+import { defaultNavigationOptions, bookDetailNavigationOptions, bookListNavigationOptions, dummyNavigationOptions, childTabsNavigationOptions, loginNavigationOptions } from '../config/navigationOptions';
```

Comment:
check that you can get all options into a single value like `* as NavigationOptions`, so this way you don't have to import each one